### PR TITLE
Endpoint for global Spotify Podcast Metadata

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

File: src/types/connector.ts [hunk 1]
```diff
@@ -58,6 +58,14 @@ export interface SpotifyAggregatePayload {
         }
     }
 }
+export interface SpotifyPodcastMetadataPayload {
```
Comment (woolfg) [inline src/types/connector.ts:61]: do we ignore the other fields?

File: db_schema/schema.sql [hunk 2]
```diff
@@ -83,6 +83,17 @@ CREATE TABLE spotifyEpisodePerformance (
   PRIMARY KEY (account_id, episode_id)
 );
 
+DROP TABLE IF EXISTS spotifyPodcastMetadata;
+CREATE TABLE spotifyPodcastMetadata (
+  account_id INTEGER NOT NULL,
+  spm_total_episodes INTEGER NOT NULL,
+  spm_starts INTEGER NOT NULL,
+  spm_streams INTEGER NOT NULL,
```
Comment (woolfg) [inline db_schema/schema.sql:91]: is INT sufficient? around 1 billion, I guess it is ok, right?

**Review by woolfg — APPROVED** [review]

File: src/types/connector.ts (hunk 1 above)
Comment (mre) [reply src/types/connector.ts:61]: yes, it's just stuff like the podcast title and description. Not really helpful for our use-case.

File: db_schema/schema.sql (hunk 2 above)
Comment (mre) [reply db_schema/schema.sql:91]: If not we have a luxury problem. 🤑

**Pull request merged**